docs/user/site.rst

```diff
@@ -235,15 +235,19 @@ mesh
     in the *batman_adv* section:
 
     The optional value *gw_sel_class* sets the gateway selection class. The
-    default is class 20, which is based on the link quality (TQ) only; class 1
+    default is class ``20``, which is based on the link quality (TQ) only; class ``1``
     is calculated from both the TQ and the announced bandwidth.
+
+    The optional value *routing_algo* allows to set up ``BATMAN V``-based meshes. 
+    If unset, the routing algorithm will default to ``BATMAN IV``.
     ::
 
       mesh = {
         vxlan = true,
         filter_membership_reports = false,
         batman_adv = {
           gw_sel_class = 1,
+          routing_algo = 'BATMAN_V',
```

Comment:
Please set this to `BATMAN_IV`, as this is the current default. If people blindly copy from the documentation and have no clue what they're doing that is probably not what they're expecting. Instead follow the principle of least surprise.